sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2472,30 +2466,31 @@
       }
     },
     "Twin": {
-      "description": "Twin Representation",
+      "description": "Twin Representation of the device/ module state information. Implicitly created and deleted when the corresponding device/ module identity is created or deleted in IoT Hub.",
       "type": "object",
       "properties": {
         "deviceId": {
-          "description": "The deviceId uniquely identifies the device in the IoT hub's identity registry. A case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars + {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
+          "description": "The unique identifier of the device in the IoT hub's identity registry. It is a case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars, and the following special characters {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
           "type": "string"
         },
         "moduleId": {
-          "description": "Gets and sets the Module Id.",
+          "description": "The unique identifier of the module in the IoT hub's identity registry. It is a case-sensitive string (up to 128 char long) of ASCII 7-bit alphanumeric chars, and the following special characters {'-', ':', '.', '+', '%', '_', '#', '*', '?', '!', '(', ')', ',', '=', '@', ';', '$', '''}.",
           "type": "string"
         },
         "tags": {
-          "description": "A JSON document read and written by the solution back end. Tags are not visible to device apps.",
+          "description": "A collection of key-value pairs read and written by the solution back end. They are not visible to device apps.",
```

Comment:
They are not visible to device apps? Can you explain this? I am little confused as I saw an API earlier about replacing twin tags and desired properties

Reply:
What this means is that when you do a getTwin() on the device client, you do not see the values for tags in the twin.

Reply:
I see, should we be more specific here then? It might be confusing otherwise

Reply:
This is the documentation for reference: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins#device-twins

Of course, if it is confusing, we should reword it! Any suggestions?

One way is to define the consistent rule in referring to service-client-using applications and device-client-using applications - this might help reduce the confusion when you read terms like "solution back-end" and "device-apps"